spec/models/post_spec.rb

```diff
@@ -2,24 +2,20 @@
 
 RSpec.describe Post, type: :model do
   context 'validation' do
```

Comment:
What is the purpose of these tests?

Reply:
To check if the post is still saved when some of the fields are blank.

Reply:
Nothing in the expectation indicates that the fixture has blank fields and that this is what your are testing. You should name your test and your fixture more explicitly.

`expect(FactoryBot.build(:post_with_missing_optional_fields).save).to eq(true)`

Reply:
Thanks for the feedback. I'll change this and look through the other tests too.